opm/output/eclipse/EclFilesComparator.hpp

```diff
@@ -0,0 +1,77 @@
+/*
+   Copyright 2016 Statoil ASA.
+
+   This file is part of the Open Porous Media project (OPM).
+
+   OPM is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   OPM is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+   */
+
+#ifndef ECLFILESCOMPARATOR_HPP
+#define ECLFILESCOMPARATOR_HPP 
+
+#include <vector>
+#include <opm/common/ErrorMacros.hpp>
+
+struct ecl_file_struct;
+typedef struct ecl_file_struct ecl_file_type;
+struct ecl_grid_struct;
+typedef struct ecl_grid_struct ecl_grid_type;
+
+enum eclFileEnum {RESTARTFILE, INITFILE};
+
+
+
+
+struct Deviation {
+    double abs = -1;
+    double rel = -1;
+};
+
+
+
+
+class ECLFilesComparator {
```

Comment:
this is a public interface. i expect doxy on at least the part exposed to the world.